### PR TITLE
Adds Title extension.

### PR DESCRIPTION
**Pull request by manovotny**
## Description

Vercel's<sup>*</sup> [title.sh](https://title.sh) for Raycast.

Correctly capitalizes your titles as per [The Chicago Manual of Style](http://www.chicagomanualofstyle.org/home.html).

<sub>_\* Not affiliated with Vercel._</sub>

## Screencast

https://user-images.githubusercontent.com/446260/201424106-889f9a9a-965f-4496-97fe-ddd0a123aeed.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/title/package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "title",
+  "title": "Title",
```
Comment (pernielsentikaer) [inline extensions/title/package.json:4]: A bit too generic title, `Title` doesn't really give anything, do you have any ideas?

File: extensions/title/package.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  "icon": "icon.png",
+  "author": "manovotny",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "capitalize",
+      "title": "Capitalize",
```
Comment (pernielsentikaer) [inline extensions/title/package.json:15]: Same here (since it don't Capitalize) 😊

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you 😊

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by manovotny** [comment]
@pernielsentikaer thanks for the feedback! I can see where it's a bit too generic. I'd like to ideate out loud together before making any changes so we can come to an agreement and get on the same page.

I guess I see two options:

1. Make it more hyper specific by calling it `title.sh` or `The Chicago Manual of Style Titles`.
2. Add `The Chicago Manual of Style` or `title.sh` to the [existing Change Case extension](https://www.raycast.com/es183923/change-case).

Thoughts?

**Comment by pernielsentikaer** [comment]
How different is it from the current `titleCase` in change-case?

**Comment by manovotny** [comment]
The Chicago Manual of Style can have slightly different rules. Here's an example below. Notice the word `some`.

CleanShot 2022-11-30 at 09 54 09@2x

**Comment by pernielsentikaer** [comment]
Ah, yes, that's not the same at all.

@es183923, what do you think about adding that? It might require another library to add it, though.

**Comment by pernielsentikaer** [comment]
Giving a last ping @es183923 😊

**Comment by pernielsentikaer** [comment]
Sorry for missing this @manovotny - what is your default use case for this scenario?

Does it match how the `usecase extension` is doing it, or?

**Comment by manovotny** [comment]
> Sorry for missing this @manovotny - what is your default use case for this scenario?
> 
> Does it match how the `usecase extension` is doing it, or?

@@pernielsentikaer not quite, but this PR could be changed to act like it. As you can see in the video above, it was expecting text to be typed into an input vs. being in the clipboard already. That is the only difference, but it could be changed to act exactly like Change Case. Or again, a completely separate plugin. Totally your call.

**Comment by pernielsentikaer** [comment]
What about a separate extension with two commands, input and one with (clipboard/selected text) - this and a less generic title/description would do it for sure 💯

**Comment by es183923** [comment]
i suggest a separate extension, emphasizing on that it follows the chicago format

**Comment by pernielsentikaer** [comment]
Can we update the title and maybe also the logo to not just use the same as the official [Vercel extension](https://www.raycast.com/vercel/vercast)

**Pull request closed without merging**